rhobs/alerting/data_plane/prometheus.latency_component_onboarding_alerts.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-latency-component-onboarding-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: latency-component-onboarding
+    interval: 1m
+    rules:
+    - alert: LatencyComponentOnboarding
+      expr: |
+        (
+          (
+            increase(redhat_appstudio_buildservice_component_onboarding_time_bucket{le="+Inf"}[5m])
+            - ignoring(le)
+            increase(redhat_appstudio_buildservice_component_onboarding_time_bucket{le="60"}[5m])
+          ) / ignoring(le)
+          increase(redhat_appstudio_buildservice_component_onboarding_time_bucket{le="+Inf"}[5m])
+        ) > 0.10
+      for: 1m
+      labels:
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
+          Latency of component onboarding time exceeded
+        description: >
+          Time spent from the moment of Component creation till the initial simple build is submitted,
+          if PaC provision is requested upon the Component creation, then till the provision finishes has been over
+          60s for more than 10% of requests during the last 5 minutes on cluster
+          {{ $labels.source_cluster }}
```

Please include the actual expression value in each description.
```suggestion
          Time spent from the moment of Component creation till the initial simple build is submitted,
          if PaC provision is requested upon the Component creation, then till the provision finishes has been over
          60s for {{ value | humanizePercentage }} of requests during the last 5 minutes on cluster
          {{ $labels.source_cluster }}
```